Jupyter Notebooks/LUVOIR/asahoo_merging_1.ipynb

```diff
@@ -0,0 +1,502 @@
+    "fig = plt.figure(figsize=(20,10))\n",
+    "for i in range(0, nth_lo):\n",
+    "    plt.subplot(3,5,i+1)\n",
+    "    im = plt.imshow(np.log(np.abs(np.reshape(G_LO_real[i], [227,227]))), vmin=-30, vmax=-5)\n",
+    "    plt.title(\"G_LO_real_%d\"%i)\n",
+    "    plt.colorbar(im)\n",
+    "plt.tight_layout()\n",
+    "\n",
+    "\n",
+    "fig = plt.figure(figsize=(20,10))\n",
+    "for i in range(0, nth_lo):\n",
+    "    plt.subplot(3,5,i+1)\n",
+    "    im = plt.imshow(np.log(np.abs(np.reshape(G_LO_imag[i], [227,227]))), vmin=-30, vmax=-5)\n",
+    "    plt.title(\"G_LO_imag_%d\"%i)\n",
+    "    plt.colorbar(im)\n",
+    "plt.tight_layout()\n",
+    "\n",
+    "\n",
+    "# print(\"\\n\",\"*****************      MID_ORDER field imaginary and real part    **************\",\"\\n\")\n",
+    "# fig = plt.figure(figsize=(40,20))\n",
+    "# for i in range(0, nth_mid):\n",
+    "#     plt.subplot(20,36,i+1)\n",
+    "#     im = plt.imshow(np.log(np.abs(np.reshape(G_MID_real[i], [227,227]))), vmin=-30, vmax=-5)\n",
+    "#     plt.title(\"G_MID_real_%d\"%i)\n",
+    "#     plt.colorbar(im)\n",
+    "# #plt.tight_layout()\n",
+    "\n",
+    "\n",
+    "# fig = plt.figure(figsize=(40,20))\n",
+    "# for i in range(0, nth_mid):\n",
+    "#     plt.subplot(20,36,i+1)\n",
+    "#     im = plt.imshow(np.log(np.abs(np.reshape(G_MID_imag[i], [227,227]))), vmin=-30, vmax=-5)\n",
+    "#     plt.title(\"G_MID_imag_%d\"%i)\n",
+    "#     plt.colorbar(im)\n",
+    "# plt.tight_layout()\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "simplified-italy",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "design = 'medium'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "controlled-storage",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "overall_dir = util.create_data_path(CONFIG_PASTIS.get('local', 'local_data_path'), telescope = 'luvoir-'+design)\n",
+    "os.makedirs(overall_dir, exist_ok=True)\n",
+    "resDir = os.path.join(overall_dir, 'matrix_numerical')\n",
+    "\n",
+    "# # Moving parts parameters\n",
+    "max_LO = CONFIG_PASTIS.getint('calibration', 'max_LO')\n",
+    "max_MID = CONFIG_PASTIS.getint('calibration', 'max_MID')\n",
+    "max_HI = CONFIG_PASTIS.getint('calibration', 'max_HI')\n",
+    "num_DM_act = CONFIG_PASTIS.getint('calibration', 'num_DM_act')\n",
+    "\n",
+    "# # General telescope parameters\n",
+    "nb_seg = CONFIG_PASTIS.getint('LUVOIR', 'nb_subapertures')\n",
+    "wvln = CONFIG_PASTIS.getfloat('LUVOIR', 'lambda') * 1e-9  # m\n",
+    "diam = CONFIG_PASTIS.getfloat('LUVOIR', 'diameter')  # m\n",
+    "nm_aber = CONFIG_PASTIS.getfloat('calibration', 'single_aberration') * 1e-9   # m\n",
+    "\n",
+    "# # Image system parameters\n",
+    "im_lamD = CONFIG_PASTIS.getfloat('numerical', 'im_size_lamD_hcipy')  # image size in lambda/D\n",
```

Comment:
these are likely things you won't need in the notebook either, as they are handled by the simulator directly

Reply:
ok got it.